moodlecli/aws.py

```diff
@@ -0,0 +1,43 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+
+
+def create_bucket_if_doesnt_exist(bucket_name):
+    """Create an S3 bucket in a specified region
+
+    :param bucket_name: Bucket to create
+    :param region: String region to create bucket in, e.g., 'us-west-2'
+    :return: True if bucket created, else False
+    """
+    try:
+        s3_client = boto3.client('s3')
+        return s3_client.create_bucket(Bucket=bucket_name)
+    except ClientError as e:
+        print(e)
+        return False
+    return True
+
+
+def send_moodle_sudent_grades_to_bucket(data, bucket_name, key):
```

Comment:
Minor: The name of this function may be overly specific since it could be used for various data objects. Maybe something like `put_json_data` instead?